src/network/network_gui.cpp

```diff
@@ -1628,10 +1628,11 @@ static const NWidgetPart _nested_client_list_widgets[] = {
 		EndContainer(),
 		NWidget(NWID_HORIZONTAL),
 			NWidget(WWT_MATRIX, COLOUR_GREY, WID_CL_MATRIX), SetMinimalSize(180, 0), SetResize(1, 1), SetFill(1, 1), SetMatrixDataTip(1, 0, STR_NULL), SetScrollbar(WID_CL_SCROLLBAR),
-			NWidget(NWID_VERTICAL),
-				NWidget(NWID_VSCROLLBAR, COLOUR_GREY, WID_CL_SCROLLBAR),
-				NWidget(WWT_RESIZEBOX, COLOUR_GREY),
-			EndContainer(),
+			NWidget(NWID_VSCROLLBAR, COLOUR_GREY, WID_CL_SCROLLBAR),
+		EndContainer(),
+		NWidget(NWID_HORIZONTAL),
+			NWidget(WWT_TEXTBTN, COLOUR_GREY, WID_CL_CLIENT_NUMBER), SetFill(1, 0), SetMinimalTextLines(1, 0), SetResize(1, 0), SetAlignment(SA_RIGHT),
```

Comment:
Shouldn't this be a `WWT_LABEL` or maybe `WWT_TEXT`? It's not a button you are supposed to click on, right?

Reply:
It was my intend to draw the container of the count of clients/companies as a box shape.
Anyway I changed ``WWT_TEXTBTN`` into ``WWT_TEXT`` now, though it looks weird for me.
Please let me know if there is a better idea.

Reply:
It definitely looks different, though I think it looking different is not a bad thing as it makes it more clear it is not one of the lines in the table, but rather a footer. With the text button it looked like it was part of the table.
Having said that, I think it looks better when the string is aligned in the center (`SA_CENTER`), as the right alignment looks weird to me. Though, that can also be my bad taste. Maybe some others have an opinion they want to share about the looks of the button and alignment of the text.

Reply:
I agree that right-alignment looks strange here. It should either be centered or left-aligned, IMO. Left alignment probably makes more sense, since this is not a button.

Reply:
Okay, let's do them.

**Left:**
![image](https://user-images.githubusercontent.com/10726524/122682964-4fdf4a80-d237-11eb-8f80-0e5d78d0a3cd.png)

**Centre:**
![image](https://user-images.githubusercontent.com/10726524/122683096-178c3c00-d238-11eb-9e3e-8c438b6d1cd7.png)

I like whatever and so I can't decide it.

Reply:
Center is not bad, left is OK too, both are better than right. If I have to decide I'd say center.
BTW it seems the matrix is longer than the scroll bar.

Reply:
Center looks really nice in my book :D

Reply:
OK, now make it centered.